src/clr-addons/treetable/treetable.scss

```diff
@@ -241,7 +242,7 @@
         width: auto;
 
         .treetable-row:hover {
-          background-color: var(--clr-datagrid-row-hover);
+          background-color: $clr-highlight-neutral-200;
```

Comment:
same here just use the variable --clr-datagrid-row-hover-color and overwrite it's value to this